.github/workflows/rust.yml

```diff
@@ -353,7 +353,7 @@ jobs:
         if: env.BACKEND_CHANGED == 'true'
         uses: dtolnay/rust-toolchain@stable
         with:
-          toolchain: 1.88
+          toolchain: 1.91
```

Comment:
I don't know which version we should use in CI. Our minimum supported version is 1.89, so shouldn't we use that to ensure everything works in 1.89? But that way we don't get all the new lints.